assets/js/blocks/active-filters/block.json

```diff
@@ -1,13 +1,14 @@
 {
 	"name": "woocommerce/active-filters",
 	"version": "1.0.0",
-	"title": "Active Product Filters",
+	"title": "Active Product Filter Controls",
 	"description": "Display the currently active product filters.",
 	"category": "woocommerce",
 	"keywords": [ "WooCommerce" ],
 	"supports": {
 		"html": false,
 		"multiple": false,
```

Comment:
We're losing the ability to limit the number of Active Filters, Price Filter, and Stock Filter blocks on a page.